files/image_config/hostcfgd/hostcfgd

```diff
@@ -219,6 +220,537 @@ class AaaCfg(object):
+        try:
+            subprocess.check_call(cmd, shell=True)
+            appIntf['admin_status'] = admin_status
+        except subprocess.CalledProcessError as err:
+            syslog.syslog(syslog.LOG_ERR, "{} - failed: return code - {}, output:\n{}"
+                    .format(err.cmd, err.returncode, err.output))
+
+    def mgmt_intf_description_set (self, ifname, description, appIntf):
+        syslog.syslog(syslog.LOG_INFO, "Configure description, cmd - {}".format(description))
+        appIntf['description'] = description
+
+    def mgmt_intf_speed_set (self, ifname, speed, autoneg, appIntf):
+        duplex = "full"
+        cmd = 'ethtool -s {} speed {} duplex {} autoneg {}; ifup {}'.format(ifname, speed, duplex, autoneg, ifname)
+        syslog.syslog(syslog.LOG_INFO, "Configure SPEED, cmd - {}".format(cmd))
+
+        try:
+            subprocess.check_call(cmd, shell=True)
+            appIntf['speed'] = speed
+        except subprocess.CalledProcessError as err:
+            syslog.syslog(syslog.LOG_ERR, "{} - failed: return code - {}, output:\n{}"
+                    .format(err.cmd, err.returncode, err.output))
+
+    def mgmt_intf_autoneg_set (self, ifname, autoneg, appIntf):
+        an = "on"
+        if autoneg == "true":
+            an = "on"
+        elif autoneg == "false":
+            an = "off"
+
+        cmd = "ethtool -s {} autoneg {}".format(ifname, an)
+        syslog.syslog(syslog.LOG_INFO, "Configure autoneg, cmd - {}".format(cmd))
+
+        try:
+            subprocess.check_call(cmd, shell=True)
+            appIntf['autoneg'] = autoneg
+        except subprocess.CalledProcessError as err:
+             syslog.syslog(syslog.LOG_ERR, "{} - failed: return code - {}, output:\n{}".format(err.cmd, err.returncode, err.output))
+
+    def mgmt_intf_gwaddr_set(self, ifname, ip_prefix, gwaddr, op):
+        if not self.mgmt_vrf_cfg.handle_mgmt_intf_gwaddr_cfg(op, ifname, gwaddr):
+            try:
+                ip_conf = netaddr.IPNetwork(ip_prefix)
+                ver = "-4"
+                if ip_conf.version == 6:
+                    ver = "-6"
+
+                cmd = "ip {} route {} {}/{} via {} dev {}".format(ver, op.lower(), ip_conf.network, ip_conf.prefixlen, gwaddr, ifname)
+
+                syslog.syslog(syslog.LOG_INFO, "Configure gwaddr , cmd : {}".format(cmd))
+                subprocess.check_call(cmd, shell=True)
+            except subprocess.CalledProcessError as err:
+                syslog.syslog(syslog.LOG_ERR, "{} - failed: return code - {}, output:\n{}".format(err.cmd, err.returncode, err.output))
+                pass
+
+    def mgmt_intf_default_gwaddr_set (self, ifname, default_gw, op):
+        try:
+            ver = "-4"
+            mgmt_conf = netaddr.IPNetwork(default_gw)
+            if mgmt_conf.version == 6:
+                ver = "-6"
+
+            cmd = "ip {} route {} default via {} dev {}".format(ver, op.lower(), default_gw, ifname)
```

Comment:
So this must be to the `default `table and here it looks like adding to main table.  Refer https://github.com/Azure/sonic-buildimage/blob/master/files/image_config/interfaces/interfaces.j2#L76

Reply:
addressed used default/5000 tables as per the vrf config